src/domains/transaction/pages/SendIpfs/SendIpfs.tsx

```diff
@@ -90,8 +90,12 @@ export const SendIpfs = () => {
 		};
 
 		try {
-			const abortSignal = abortRef.current?.signal;
+			if (activeWallet.isLedger()) {
```

Comment:
Why do we need to manually connect here? The SDK will connect when it actually needs a connection and not otherwise, see https://github.com/ArkEcosystem/platform-sdk/blob/master/packages/platform-sdk-ark/source/transaction.service.ts#L219-L220.

Reply:
Yes actually we need to call connect from `useLedgerConnection` https://github.com/ArkEcosystem/desktop-wallet/blob/develop/src/app/contexts/Ledger/hooks/connection.ts#L52 because it updates the internal state that is necessary for the ledger review step